static/js/issuesTable.js

```diff
@@ -1,35 +1,86 @@
 const IssuesTable = {
-    props: ['engagement', 'engagementsList'],
+    props: ['engagement'],
     components: {
         'board-creation-modal': BoardCreationModal,
         'ticket-creation-modal': TicketCreationModal,
         'filter-toolbar-container': FilterToolbarContainer,
+        'small-issues-table': SmallIssuesTable,
+        'ticket-view-container': TicketViewContainer,
     },
     data() {
         return {
             url: issues_api_url,
             issues_url: issues_api_url,
             noTicketSelected: true,
             preFilterMap: {},
-            table_id: "#issues-table"
+            table_id: "#issues-table",
+            selectedTicket: null,
+            pageNumber: 1,
+            maxPageCount: 1,
+
+            all_users: [],
         }
     },
     mounted(){
         this.setTableCheckEvents()
+        $(this.table_id).on('click-row.bs.table', (e, row, $el, field) => {
```

Comment:
a small note: `$(this.table_id)` can be stored as a const nad re-used in function scope. This makes the jquery selector run only once